bin/mksetup

```diff
@@ -68,6 +68,7 @@ if ("$?EUPS_DIR" == "1" ) then
 else
    setenv EUPS_DIR "%(eupsdir)s"
 endif
+set eupslocalpath = `echo "$EUPS_DIR" | perl -pe "s/ /-+-/"`
```

Comment:
Could you `sed` this instead? I'm trying to ultimately get rid of `perl` as a dependency...